core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -1367,6 +1367,18 @@ private SqlLibraryOperators() {
           OperandTypes.STRING_STRING,
           SqlFunctionCategory.NUMERIC);
 
+  @LibraryOperator(libraries = {SNOWFLAKE, BIG_QUERY})
```

Comment:
We should include only BigQuery in the Libraries, because this function definition does not align with the behavior of Snowflake as it accepts up-to 7 arguments and not just 4.

Reply:
Done.